### PR TITLE
samples: Change samples to sample

### PR DESCRIPTION
**Pull request by katgiadla**
Some names of the testcases samples do not start with "sample".
These testcases rename from "samples" to "sample".

Signed-off-by: Katarzyna Giadla <katarzyna.giadla@nordicsemi.no>

test-sdk-nrf: sdk-nrf-PR-7290

### CODE REVIEW

**Review by coderbyheart — APPROVED** [review]

**Review by frkv — APPROVED** [review]
LGTM (crypto)

**Review by czeslawmakarski — APPROVED** [review]

**Review by canisLupus1313 — APPROVED** [review]

**Review by simensrostad — APPROVED** [review]

**Review by plskeggs — APPROVED** [review]
LGTM (nRF Cloud)

**Review by glarsennordic — APPROVED** [review]
Changes to samples/nrf9160 LGTM

**Review by tomchy — APPROVED** [review]

**Comment by pdunaj** [comment]
please note that test cases are added all the time so after this is merged, new commits can try to use old names
@emob-nordic , @MarekPieta , @rakons , @zycz  - let's see if we can stay in sync with changes already in progress

File: samples/app_event_manager/sample.yaml [hunk 1]
```diff
@@ -3,21 +3,21 @@ sample:
...
+    type: multi_line
+  integration_platforms:
+    - qemu_cortex_m3
+    - nrf52dk_nrf52832
+    - nrf52840dk_nrf52840
+    - nrf9160dk_nrf9160_ns
+    - nrf21540dk_nrf52840
 tests:
-  samples.app_event_manager:
+  sample.event_manager:
```
Comment (zycz) [inline samples/app_event_manager/sample.yaml:22]: Is name change app_event_manager/event_manager necessary?

Comment (katgiadla) [reply samples/app_event_manager/sample.yaml]: It was misspelling. Done.

**Pull request merged**